fe/fe-core/src/main/java/com/starrocks/alter/SchemaChangeHandler.java

```diff
@@ -1105,10 +1105,12 @@ private void getAlterJobV2Infos(Database db, List<AlterJobV2> alterJobsV2,
             if (alterJob.getDbId() != db.getId()) {
                 continue;
             }
-            if (ctx != null) {
-                if (!GlobalStateMgr.getCurrentState().getAuth()
-                        .checkTblPriv(ctx, db.getFullName(), alterJob.getTableName(), PrivPredicate.ALTER)) {
-                    continue;
+            if (!GlobalStateMgr.getCurrentState().isUsingNewPrivilege()) {
```

Comment:
If I'm not mistaken, when `isUsingNewPrivilege` is true, all privilege checks will pass. Is it what you want?

Reply:
Yes, this method is only used by `show /proc` statement, and discussed with PM @wangsimo0 before, the final decision is that we won't check privilege for `show /proc` statement.